### PR TITLE
ui: add mathjax to record details page

### PR DESCRIPTION
**Pull request by anikachurilova**
* closes https://github.com/zenodo/rdm-project/issues/548

Landing page:
Screenshot 2023-11-15 at 17 07 02

Request page:
Screenshot 2023-11-15 at 17 07 40

### CODE REVIEW

File: invenio_app_rdm/config.py [hunk 1]
```diff
@@ -234,6 +234,11 @@ def _(x):
 
 THEME_JAVASCRIPT_TEMPLATE = "invenio_app_rdm/javascript.html"
 
+THEME_MATHJAX_CDN = (
```
Comment (kpsherva) [inline invenio_app_rdm/config.py:237]: was it agreed on to use the cdn in the invenioRDM core? should we use it only in zenodo? to be discussed.

File: invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html [hunk 2]
```diff
@@ -360,6 +360,7 @@ <h4>{{ _("Request access") }}</h4>
 {%- endblock page_body %}
 
 {%- block javascript %}
+  <script type="text/javascript" src="{{ config.THEME_MATHJAX_CDN }}"></script>
```
Comment (slint) [inline invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html:363]:
```suggestion
  {% if config.THEME_MATHJAX_CDN %}
  <script type="text/javascript" src="{{ config.THEME_MATHJAX_CDN }}"></script>
  {% endif %}
```

As suggested above.

Comment (slint) [reply invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html:363]: Now that I think about it, shouldn't this bee already defined like this in `invenio-theme`? Where did the `THEME_` prefix come from?

Comment (anikachurilova) [reply invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html:363]: There's  mathjax defined in `invenio-theme`. Took this constant name from the legacy https://github.com/zenodo/zenodo/blob/master/zenodo/config.py#L1427

**Pull request merged**